crypto/fipsmodule/evp/p_rsa.c

```diff
@@ -734,7 +734,7 @@ static int pkey_rsa_ctrl_str(EVP_PKEY_CTX *ctx, const char *type,
     // "max" and "auto" are not supported
     long saltlen;
 
-    if (!strcmp(value, "digest")) {
+    if (!strcmp(value, "digest") || !strcmp(value, "-1")) {
```

Comment:
A comment here about why "-1" is the same as `RSA_PSS_SALTLEN_DIGEST` would be helpful.